src/main/scala/com/mozilla/telemetry/pings/package.scala

```diff
@@ -0,0 +1,225 @@
+
+  case class Addons(activeExperiment: Option[ActiveExperiment])
+
+  case class Settings(
+                       blocklistEnabled: Option[Boolean],
+                       isDefaultBrowser: Option[Boolean],
+                       e10sEnabled: Option[Boolean],
+                       e10sCohort: Option[String],
+                       locale: Option[String],
+                       telemetryEnabled: Option[Boolean]
+                     )
+
+  case class Profile(
+                      creationDate: Option[Int],
+                      resetDate: Option[Int]
+                    )
+
+  case class Meta(
+                   Host: Option[String],
+                   Hostname: Option[String],
+                   Size: Option[Double],
+                   Timestamp: Long,
+                   Type: Option[String],
+                   appBuildId: String,
+                   appName: String,
+                   appUpdateChannel: Option[String],
+                   appVendor: Option[String],
+                   appVersion: Double,
+                   clientId: Option[String],
+                   creationTimestamp: Option[Float],
+                   docType: Option[String],
+                   documentId: Option[String],
+                   geoCity: Option[String],
+                   geoCountry: String,
+                   normalizedChannel: String,
+                   os: String,
+                   sampleId: Option[Double],
+                   sourceName: Option[String],
+                   sourceVersion: Option[Int],
+                   submissionDate: String,
+                   telemetryEnabled: Option[Boolean],
+                   // Common fields preparsed by hindsight
+                   `environment.build`: Option[EnvironmentBuild],
+                   `environment.settings`: Option[Settings],
+                   `environment.system`: Option[System],
+                   `environment.profile`: Option[Profile],
+                   `environment.addons`: Option[Addons],
+                   // Main ping fields preparsed by hindsight
+                   `payload.simpleMeasurements`: JValue,
+                   `payload.keyedHistograms`: JValue,
+                   `payload.histograms`: JValue,
+                   `payload.info`: JValue
+                 )
+
+  case class CrashPayload(
+                           crashDate: String,
+                           processType: Option[String],
+                           hasCrashEnvironment: Option[Boolean],
+                           metadata: Option[Map[String, String]],
+                           version: Option[Int]
+                         )
+
+  case class CrashPing(
+                        application: Application,
+                        clientId: Option[String],
+                        payload: CrashPayload,
+                        // Environment is omitted it's partially available under meta
+                        meta: Meta
+                      ){
+
```

Comment:
Please remove the newline. We should add a linter to travis to have an uniform style.